reference/functions/pgroonga-highlight-html.md

```diff
@@ -96,6 +96,56 @@ SELECT pgroonga_highlight_html('one two three four five',
 
 The keywords are surrounded with `<span class="keyword">` and `</span>`. `<`, `>`, `&` and `"` in `target` is HTML escaped.
 
+**Special Case: Using with `NormalizerTable`**
+
+If you specify `NormalizerTable`, like in this `CREATE INDEX USING pgroonga` document ([`How to use NormalizerTable` section](https://pgroonga.github.io/reference/create-index-using-pgroonga.html#normalizer-table)), the specified PGroonga index must have not only `TokenNgram` with `"report_source_location", true"` option but also both `Normalizer` and `NormalizerTable` with `"report_source_offset", true"` option for each.
```

Comment:
Could you stop using `` `...` `` markup to highlight words? It's a markup for code.
Could you use `[label][id]` style link here too?

```suggestion
If you specify `NormalizerTable`, like in this `CREATE INDEX USING pgroonga` document ([How to use NormalizerTable section][(https://pgroonga.github.io/reference/create-index-using-pgroonga.html#normalizer-table)), the specified PGroonga index must have not only `TokenNgram` with `"report_source_location", true"` option but also both `Normalizer` and `NormalizerTable` with `"report_source_offset", true"` option for each.
```

Reply:
I'm not good at this... Hopefully my fix will work.